src/App.js

```diff
@@ -2,16 +2,49 @@
  * Created by ttnd on 27/12/16.
  */
 import React from 'react'
+import Student from './Student';
+import HelloWorldComponent from './HelloWorldComponent';
+import Header from './Header';
+import Users from './Users';
+import BasicPage from './BasicPage';
+import UserPage from './UserPage';
 
 export default class App extends React.Component {
   constructor() {
```

Comment:
pass props as a variable to super as a best practice to avoid having this.props as undefined in constructor
constructor(props) { super(props) }